modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala

```diff
@@ -106,7 +106,7 @@ class SeqexecUIApiRoutes(auth: AuthenticationService, events: (engine.EventQueue
                     \/.fromTryCatchNonFatal(new SPObservationID(oid))
                       .fold(Task.fail, Task.now)
                 u     <- se.load(inputQueue, obsId)
-                resp  <- u.fold(_ => NotFound(s"Not found sequence $oid"), _ => Ok(SequencesQueue[SequenceId](List(oid))))
+                resp  <- u.fold(_ => NotFound(s"Not found sequence $oid"), _ => Ok(oid))
```

Comment:
Actually, I was convinced that change must have broken something, but I couldn't find anything.

In any case if we see something weird in the client that you didn't see before, this line should be the prime suspect for potential problems 🎱